app/models/project.rb

```diff
@@ -73,9 +73,10 @@ def tag_list
   end
 
   def tag_list=(names)
-    self.tags = names.split(",").map do |n|
+    self.tags = names.split(/\s*,\s*/).compact.reject(&:empty?).uniq.map do |n|
       Tag.where(name: n.strip).first_or_create!
     end
+    puts(self.tags)
```

Comment:
If this is not required here, can this be removed?

Reply:
oops forgot by mistake, it's removed now